tutorials/notebooks/mct_features_notebooks/pytorch/example_pytorch_post_training_quantization.ipynb

```diff
@@ -0,0 +1,470 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7cf96fb4",
+   "metadata": {
+    "id": "7cf96fb4"
+   },
+   "source": [
+    "# Post-Training Quantization in PyTorch using the Model Compression Toolkit (MCT)\n",
+    "[Run this tutorial in Google Colab](https://colab.research.google.com/github/sony/model_optimization/blob/main/tutorials/notebooks/mct_features_notebooks/pytorch/example_pytorch_post_training_quantization.ipynb)\n",
+    "\n",
+    "## Overview\n",
+    "This quick-start guide explains how to use the **Model Compression Toolkit (MCT)** to quantize a PyTorch model. We will load a pre-trained model and  quantize it using the MCT with **Post-Training Quatntization (PTQ)**. Finally, we will evaluate the quantized model and export it to an ONNX file.\n",
+    "\n",
+    "## Summary\n",
+    "In this tutorial, we will cover:\n",
+    "\n",
+    "1. Loading and preprocessing ImageNet’s validation dataset.\n",
+    "2. Constructing an unlabeled representative dataset.\n",
+    "3. Hardware-Friendly Post-Training Quantization using MCT.\n",
```

Comment:
I think we can remove the "Hardware-friendly" term from this tutorial, the weights thresholds are no longer POT (in the default TPC) and this is not important in the scope of the tutorial and its purpose.